package.json

```diff
@@ -36,6 +36,7 @@
     "@iota/core": "1.0.0-beta.30",
     "@iota/http-client": "1.0.0-beta.30",
     "@iota/mam": "^0.7.3",
```

Comment:
Please remove the legacy mam lib once the refactor is completed

Reply:
Also if there is no need in `@iota/http-client` and other legacy libs, please remove them as well